src/ConcreteOperations/issubset.jl

```diff
@@ -152,15 +152,20 @@ function ⊆(H1::AbstractHyperrectangle{N},
            witness::Bool=false
           )::Union{Bool, Tuple{Bool, Vector{N}}} where {N<:Real}
     @assert dim(H1) == dim(H2)
-
-    for i in 1:dim(H1)
-        c_dist = center(H1)[i] - center(H2)[i]
+    zero_N = zero(N)
```

Comment:
We used to do that in the past, but the compiler will inline `zero(N)` anyway and so I prefer to remove this variable.

Reply:
Ok :+1: